central/vulnmgmt/vulnerabilityrequest/datastore/datastore_impl_test.go

```diff
@@ -78,10 +79,11 @@ func TestVulnRequestDataStore(t *testing.T) {
 type VulnRequestDataStoreTestSuite struct {
 	suite.Suite
 
-	mockCtrl     *gomock.Controller
-	mockStore    *storeMock.MockStore
-	mockSearcher *searcherMock.MockSearcher
-	mockIdentity *mockIdentity.MockIdentity
+	mockCtrl           *gomock.Controller
```

Comment:
It would be great if we could merge this one after #14382 such that the tests work on real data and not pointless mock everything.